factory/factory.go

```diff
@@ -43,14 +42,11 @@ func InitConfigFactory(f string) error {
 		if NssfConfig.Configuration.WebuiUri == "" {
 			NssfConfig.Configuration.WebuiUri = "webui:9876"
 		}
-		roc := os.Getenv("MANAGED_BY_CONFIG_POD")
-		if roc == "true" {
+		if os.Getenv("MANAGED_BY_CONFIG_POD") == "true" {
 			logger.CfgLog.Infoln("MANAGED_BY_CONFIG_POD is true")
-			commChannel := client.ConfigWatcher(NssfConfig.Configuration.WebuiUri)
-			go NssfConfig.updateConfig(commChannel)
 		} else {
 			go func() {
-				logger.CfgLog.Infoln("Use helm chart config ")
+				logger.CfgLog.Infoln("use helm chart config")
 				ConfigPodTrigger <- true
 			}()
 		}
```

Comment:
I think this `if` statement has to be move into the `service/init.go` (similarly as it was done in the `amf` https://github.com/omec-project/amf/pull/330/files#diff-3289814d5eb2b5ebf4f9002fc41f602d2af395ca60676275f6113c0defa8f485)

Reply:
Done